tests/integration/pause.tcl

```diff
@@ -76,7 +77,8 @@ start_server {tags {"external:skip"}} {
         common_bench_setup $cmd1
         
         # Configure maxmemory to be tightly upper-bound to DB size
-        r config set maxmemory [expr [s used_memory] + 100000] 
+        set maxmemory [expr [s used_memory] + 100000]
+        r config set maxmemory $maxmemory 
```

Comment:
not sure what are the test suite changes that came with the last commit.
seems like WIP. but i guess that eventually, we want to add a test that challenges that (and explodes if we didn't remove the read handler)